### PR TITLE
Support INCITE checkpoints

### PR DESCRIPTION
**Pull request by carmocca**
https://www.together.xyz/blog/redpajama-models-v1

For the howto, do we want to add an `INCITE.md` page or merge it with `pythia.md` (#19) into a `checkpoints.md` file?

### CODE REVIEW

File: lit_stablelm/model.py [hunk 1]
```diff
@@ -63,6 +64,67 @@ def from_name(cls, name: str) -> Self:
     "pythia-6.9b": dict(block_size=2048, n_layer=32, n_embd=16384, n_head=32, padding_multiple=128),
     # https://huggingface.co/EleutherAI/pythia-12b/blob/main/config.json
     "pythia-12b": dict(block_size=2048, n_layer=36, n_embd=20480, n_head=40, padding_multiple=128),
+    # togethercomputer
```
Comment (carmocca) [inline lit_stablelm/model.py:67]: This dictionary has gotten quite long. I'll split this into a `configs.py` file in a separate PR

File: chat.py
Comment (lantiga) [inline chat.py:1]: This only supports StableML right?

**Review by lantiga — APPROVED** [review]
Looks great!
Let’s keep howtos in separate files for the different models for discoverability.

Also in the howtos let’s add a sentence describing what the models are, where they come from and by whom

File: chat.py
Comment (carmocca) [reply chat.py:1]: All models actually! The prompt config is defined in https://github.com/Lightning-AI/lit-stablelm/pull/24/files#diff-bf2e2867f177a800d3a98c40572729434405fe4e303ac655a9c4f78ddaf3ac90R141

Comment (lantiga) [reply chat.py:1]: Oh right!

**Pull request merged**